### ISSUE TITLE
Schedule with payload of 0 does not work

### ISSUE THREAD

**Issue by Simon01011** [Steve-Mcl/node-red-contrib-cron-plus#40]
Hi!

Trying to set a payload of 0 (number, not string) for my schedule does not work, it is always reset to "payload" (string)

Flow to reproduce:
```
[{"id":"30fba414.d520fc","type":"debug","z":"a2720ba6.4a7be8","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"true","targetType":"full","statusVal":"","statusType":"auto","x":510,"y":380,"wires":[]},{"id":"df17949b.5dcd28","type":"cronplus","z":"a2720ba6.4a7be8","name":"","outputField":"payload","timeZone":"","persistDynamic":false,"commandResponseMsgOutput":"output1","outputs":1,"options":[],"x":340,"y":380,"wires":[["30fba414.d520fc"]]},{"id":"c1985a5d.26a1c8","type":"inject","z":"a2720ba6.4a7be8","name":"test with 1","props":[{"p":"payload"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","payload":"[{\"name\":\"test with 1\",\"expression\":\"0 32 14 * * 1,2,3,4,5 *\",\"expressionType\":\"cron\",\"payloadType\":\"default\",\"solarEvents\":\"sunrise\",\"solarType\":\"selected\",\"location\":\"54.999320540937035 -1.417407989501953\",\"offset\":\"0\",\"limit\":null,\"payload\":1,\"command\":\"add\",\"topic\":\"\",\"type\":\"date\"}]","payloadType":"json","x":140,"y":380,"wires":[["df17949b.5dcd28"]]},{"id":"d13dc99f.92d288","type":"inject","z":"a2720ba6.4a7be8","name":"test with 0","props":[{"p":"payload"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","payload":"[{\"name\":\"test with 0\",\"expression\":\"0 33 14 * * 1,2,3,4,5 *\",\"expressionType\":\"cron\",\"payloadType\":\"default\",\"solarEvents\":\"sunrise\",\"solarType\":\"selected\",\"location\":\"54.999320540937035 -1.417407989501953\",\"offset\":\"0\",\"limit\":null,\"payload\":0,\"command\":\"add\",\"topic\":\"\",\"type\":\"date\"}]","payloadType":"json","x":140,"y":420,"wires":[["df17949b.5dcd28"]]},{"id":"64849ee8.dd85f","type":"inject","z":"a2720ba6.4a7be8","name":"remove-all","props":[{"p":"payload"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","payload":"{\"command\":\"remove-all\"}","payloadType":"json","x":140,"y":460,"wires":[["df17949b.5dcd28"]]},{"id":"637c3fae.26e7f","type":"inject","z":"a2720ba6.4a7be8","name":"list-all","props":[{"p":"payload"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","payload":"{\"command\":\"list-all\"}","payloadType":"json","x":130,"y":500,"wires":[["df17949b.5dcd28"]]}]
```
Any help is appreciated. Thank you

**Comment by Steve-Mcl** [comment]
Hi, thanks for raising this. I'll take a look this morning. Should be an easy fix.

**Comment by Steve-Mcl** [comment]
Hi, I have published a fix as Version 0.5.3 

Here is a working demo flow...

```
[{"id":"64849ee8.dd85f","type":"inject","z":"4b3f21a3.ba434","name":"remove-all","props":[{"p":"payload"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","payload":"{\"command\":\"remove-all\"}","payloadType":"json","x":2000,"y":280,"wires":[["df17949b.5dcd28"]]},{"id":"df17949b.5dcd28","type":"cronplus","z":"4b3f21a3.ba434","name":"","outputField":"payload","timeZone":"","persistDynamic":false,"commandResponseMsgOutput":"output1","outputs":1,"options":[],"x":2260,"y":160,"wires":[["30fba414.d520fc"]]},{"id":"c1985a5d.26a1c8","type":"inject","z":"4b3f21a3.ba434","name":"add \"test with 1\"","props":[{"p":"payload"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","payload":"[{\"name\":\"test with 1\",\"expression\":\"0 32 14 * * 1,2,3,4,5 *\",\"expressionType\":\"cron\",\"payloadType\":\"num\",\"solarEvents\":\"sunrise\",\"solarType\":\"selected\",\"location\":\"54.999320540937035 -1.417407989501953\",\"offset\":\"0\",\"limit\":null,\"payload\":1,\"command\":\"add\",\"topic\":\"\",\"type\":\"date\"}]","payloadType":"json","x":2020,"y":40,"wires":[["df17949b.5dcd28"]]},{"id":"d13dc99f.92d288","type":"inject","z":"4b3f21a3.ba434","name":"add \"test with 0\"","props":[{"p":"payload"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","payload":"[{\"name\":\"test with 0\",\"expression\":\"0 33 14 * * 1,2,3,4,5 *\",\"expressionType\":\"cron\",\"payloadType\":\"num\",\"solarEvents\":\"sunrise\",\"solarType\":\"selected\",\"location\":\"54.999320540937035 -1.417407989501953\",\"offset\":\"0\",\"limit\":null,\"payload\":0,\"command\":\"add\",\"topic\":\"\",\"type\":\"date\"}]","payloadType":"json","x":2020,"y":140,"wires":[["df17949b.5dcd28"]]},{"id":"637c3fae.26e7f","type":"inject","z":"4b3f21a3.ba434","name":"list-all","props":[{"p":"payload"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","payload":"{\"command\":\"list-all\"}","payloadType":"json","x":1990,"y":240,"wires":[["df17949b.5dcd28"]]},{"id":"e28c248ddd5eefec","type":"inject","z":"4b3f21a3.ba434","name":"","props":[{"p":"payload"},{"p":"topic","vt":"str"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"trigger","payload":"test with 1","payloadType":"str","x":2020,"y":80,"wires":[["df17949b.5dcd28"]]},{"id":"72e9782b2cef5a26","type":"inject","z":"4b3f21a3.ba434","name":"","props":[{"p":"payload"},{"p":"topic","vt":"str"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"trigger","payload":"test with 0","payloadType":"str","x":2020,"y":180,"wires":[["df17949b.5dcd28"]]},{"id":"30fba414.d520fc","type":"debug","z":"4b3f21a3.ba434","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"true","targetType":"full","statusVal":"","statusType":"auto","x":2430,"y":160,"wires":[]}]
```

**Comment by Simon01011** [comment]
Works perfectly, thanks for the quick response and fix! 👍 

Kind regards, Simon
